views/layouts/homepageLayoutNoFooter.ejs

```diff
@@ -128,24 +128,24 @@
     -->
 
   <!--SCRIPTS-->
-  <script src="/dependencies/sails.io.js"></script>
-  <script src="/dependencies/jquery-3.2.1.js"></script>
-  <script src="/dependencies/jquery-3.2.1.min.js"></script>
-  <script src="/dependencies/bootstrap/js/bootstrap.bundle.js"></script>
-  <script src="/dependencies/bootstrap/js/bootstrap.bundle.min.js"></script>
-  <script src="/dependencies/bootstrap/js/bootstrap.js"></script>
-  <script src="/dependencies/bootstrap/js/bootstrap.min.js"></script>
-  <script src="/dependencies/jquery-easing/jquery.easing.compatibility.js"></script>
-  <script src="/dependencies/jquery-easing/jquery.easing.js"></script>
-  <script src="/dependencies/jquery-easing/jquery.easing.min.js"></script>
-  <script src="/dependencies/lodash.js"></script>
-  <script src="/dependencies/sb-admin/sb-admin.js"></script>
-  <script src="/dependencies/validator.js"></script>
-  <script src="/js/app.js"></script>
-  <script src="/js/gbl/gblComments.js"></script>
-  <script src="/js/sdtdServer/chat.js"></script>
-  <script src="/js/sdtdServer/console.js"></script>
-  <script src="/js/sdtdServer/map.js"></script>
+  <script src="/dependencies/sails.io.js?v=1.17.5"></script>
+  <script src="/dependencies/jquery-3.2.1.js?v=1.17.5"></script>
+  <script src="/dependencies/jquery-3.2.1.min.js?v=1.17.5"></script>
+  <script src="/dependencies/bootstrap/js/bootstrap.bundle.js?v=1.17.5"></script>
+  <script src="/dependencies/bootstrap/js/bootstrap.bundle.min.js?v=1.17.5"></script>
+  <script src="/dependencies/bootstrap/js/bootstrap.js?v=1.17.5"></script>
+  <script src="/dependencies/bootstrap/js/bootstrap.min.js?v=1.17.5"></script>
```

Comment:
why are all these hard coded to 1.17.5?
why not just add it to the configs/views.js's locales section and then <%= everywhere

Reply:
oh cause it does a build. doing it with builds will cause it to make prs messier and stuff won't it?

Reply:
This is autogenerated by sails/grunt, anything between `<!--STYLES-->` and `<!--STYLES END-->`, same for scripts. The `sails-linker` task is responsible for that

Reply:
/tableflip
kill it with fire

its going to make releasing so messy

Reply:
Yep.. :/ The version in the repo will always lag one behind. I'll give hashes another go, if we hash the file content then this auto generated crap will only change when the assets change.